src/riak_kv_crdt.erl

```diff
@@ -507,13 +522,13 @@ eqc_test_() ->
      ?_test(?TIMED_QC(prop_binary_roundtrip()))}.
 
 prop_binary_roundtrip() ->
-    ?FORALL({_Type, Mod}, oneof(?MOD_MAP), 
+    ?FORALL({_Type, Mod}, oneof(?MOD_MAP),
```

Comment:
I turned mine off after working on some old files and making every line into the diff, despite a small change. I guess I'm indifferent about it in general (I argued that having the autosave feature was a good thing since trailing whitespace is "bad"), though it can make review harder for more complex changes since it draws the reviewers attention.